GSoC 2019 new proposal: spark-fits (AstroLab project/LAL organisation)

JulienPeloton opened the pull request:
Hi!

This PR adds a new proposal for GSoC 2019:
* Project: AstroLab
* Title: Apache Spark: migrating FITS Data Source API to V2
* Organization: LAL

I also fixed a typo in the LAL url (under Astrolab project): https was missing, and I link to the English version of the website.

JavierCVilla commented on _gsocproposals/2019/proposal_AstroLabSparkfits.md:
```diff
@@ -0,0 +1,40 @@
+---
+title: Apache Spark: migrating FITS Data Source API to V2
```
The colon (`:`) after `Apache Spark` makes Jekyll think that `Apache Spark` is an attribute so it gets confused generating the content. This can be solved by wrapping the whole title in quotes:

```suggestion
title: "Apache Spark: migrating FITS Data Source API to V2"
```

JavierCVilla commented on _gsocproposals/2019/proposal_AstroLabSparkfits.md:
```diff
@@ -0,0 +1,40 @@
+
+## Description
+
+High Energy Physics & Astrophysics experiments typically describe their data with complex data structures that are stored into structured file formats such as for example Hierarchical Data Format 5 (HDF5), ROOT files, or Flexible Image Transport System (FITS). For example, the FITS file format has been introduced more than thirty years ago with the requirement that developments to the format must ...
```
@JulienPeloton Do you think it makes sense to add a link to FITS documentation since it is the most relevant data format for this project?

JavierCVilla commented on _gsocprojects/2019/project_AstroLab.md:
The problem with `https` was indeed fixed on a previous PR, so it leads to a conflict now. Can you please solve it?

JulienPeloton commented:
thanks @JavierCVilla @agheata for spotting errors. I think I've fixed all issues. 

CI complains with the last commit - but I'm not sure this has nothing to do with what I've added.

JavierCVilla commented:
@JulienPeloton Looks good to me, there CI errors are not related to this PR.

Merging since both @agheata and @aflp91 approved this before modulo the conflicts.
